src/lambda_codebase/initial_commit/adfconfig.yml.j2

```
@@ -27,3 +27,6 @@ config:
   scm:
     auto-create-repositories: enabled
     default-scm-branch: main
+  # deployment-maps:
+    # Optional:
+    # allow-empty-target: "TRUE" # Needs to be "TRUE" or "FALSE". # Defaults to "FALSE", if not set. 
```

I suggest we add this with the default value set.

```suggestion
  deployment-maps:
    allow-empty-target: "FALSE"
    # ^ Needs to be "TRUE" or "FALSE". It defaults to "FALSE", if not set. 
```